diff --git a/Graphics/.mymake b/Graphics/.mymake
index 445d7def..36ecfaec 100644
--- a/Graphics/.mymake
+++ b/Graphics/.mymake
@@ -11,5 +11,3 @@ library+=jpeg
 # Set rpath, so we can supply the proper versions of libpng and libjpeg.
 linkFlags+=-Wl,-rpath,'$ORIGIN'
 
-# Copy the appropriate libraries to the target directory.
-postBuild+=../scripts/copy_libs.sh <output> <path|packagePath> png jpeg
